Signal/src/ViewControllers/ContactsPicker.swift

```diff
@@ -79,7 +79,8 @@ public class ContactsPicker: OWSViewController, UITableViewDelegate, UITableView
         CNContactThumbnailImageDataKey as CNKeyDescriptor,
         CNContactPhoneNumbersKey as CNKeyDescriptor,
         CNContactEmailAddressesKey as CNKeyDescriptor,
-        CNContactPostalAddressesKey as CNKeyDescriptor
+        CNContactPostalAddressesKey as CNKeyDescriptor,
+        CNContactVCardSerialization.descriptorForRequiredKeys()
```

Comment:
This probably has no effect, but it's nice to be explicit about our required keys.